Formula/bnfc.rb

```diff
@@ -20,12 +20,12 @@ class Bnfc < Formula
   depends_on "cabal-install" => [:build, :test]
   depends_on "ghc" => [:build, :test]
   depends_on "sphinx-doc" => :build
+  depends_on "agda" => :test
   depends_on "antlr" => :test
+  depends_on "bison" => :test
+  depends_on "flex" => :test
```

Comment:
Why not the system ones?

Reply:
The system ones error on the test now; they used to be fine but now Apple’s bison gives a syntax error on the intermediate `.y` file.